### ISSUE TITLE
Adjust ChA registration fields

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#4515]
Exact list to be confirmed:

- Remove personal summary
- Add organization (read only), status with organization, phone number (optional)

**Comment by dboyer** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Technovation-605271851956b1000ec2f24d/issues/Iridescent-CM/technovation-app/4515?planning-poker) with Zenhub @shaun-technovation @viviancan

**Comment by dboyer** [comment]
Confirming that the fields for chapter ambassador registration should be:

- Name
- Email
- Gender Identity
- Organization (pre-filled based on the chapter they are invited to)
- Status with Organization (choices are Employee or Volunteer)
- Job Title
- Phone Number (optional)
- Location

Remove company name and personal summary. We'll address changes to the birth date in a separate issue.

**Comment by shaun-technovation** [comment]
This one is on Preview, but no testing should be done until I wrap up #4543 and #4548, which will address the "Chapter Admin Activity" currently being broken.

**Comment by shaun-technovation** [comment]
This is on Preview, the fields should now be updated in all areas - registration, admin, and ChA.

**Comment by dboyer** [comment]
Confirming on preview that the following fields were available in the registration process:

- Name
- Email
- Gender Identity
- Organization (pre-filled based on the chapter they are invited to)
- Status with Organization (choices are Employee or Volunteer)
- Job Title
- Phone Number (optional)
- Location

And company name and personal summary were removed
